lldp_util.c

```diff
@@ -1311,21 +1311,21 @@ int get_arg_val_list(char *ibuf, int ilen, int *ioff,
 					*(argvallens+i) = argvalue_len;
 				}
 			} else {
-				free(arglens);
-				free(argvallens);
-				return 0;
+				numargs = 0;
+				goto err_out;
 			}
 		} else {
-			free(arglens);
-			free(argvallens);
-			return 0;
+			numargs = 0;
+			goto err_out;
 		}
 	}
 	numargs = i;
 	for (i = 0; i < numargs; i++) {
 		args[i][*(arglens+i)] = '\0';
 		argvals[i][*(argvallens+i)] = '\0';
 	}
+
+err_out:
```

Comment:
And here.

Reply:
Small nitpick, but this one is still labeled as err_out on the common return path.

Reply:
Ah, I didn't notice this is for lldp_utils.c. Sorry. Fixed now.